components/badge.css

```diff
@@ -0,0 +1,60 @@
+.badge {
+    text-align: center;
+    padding: 0.3rem;
+    font-size: 0.9rem;
+    border-radius: 0.7rem;
+    margin:1rem;
+    font-weight: bold;
+}
+
+.badge-primary {
+    color: var(--txtPrimary);
+    background-color: var(--bgPrimary);
+    border: 1px solid var(--txtPrimary);
+}
+
+.badge-secondary {
+   
+    color: var(--txtSecondary);
+    background-color: var(--bgSecondary);
+    border: 1px solid var(--txtSecondary);
+}
+
+.badge-failure {
+    color: var(--txtFailure);
+    background-color: var(--bgFailure);
+    border: 1px solid var(--txtFailure);
+}
+.badge-light{
+    color: var(--txtLight);
+    background-color: var(--bgLight);
+    border: 1px solid var(--txtLight);
+}
+.badge-dark {
+    color: var(--txtDark);
+    background-color: var(--bgDark);
+}
+/* for badges with icons */
+.badge-icon {
```

Comment:
Try to align the items in the center.

Reply:
I have added .badge class in badge-icon in html to center it